fuel/app/themes/default/partials/widget/create.php

```diff
@@ -49,6 +49,7 @@
 			<a id="returnLink" href="{{ returnUrl }}">&larr;Return to {{ returnPlace }}</a>
 			<a ng-click="showQsetHistoryImporter()">Save History</a>
 			<a id="importLink" ng-click="showQuestionImporter()">Import Questions...</a>
+			<a ng-click="showQuestionImporter2()">QI React></a>
```

Comment:
Once review is done and we're ready to commit to releasing this, make sure this tag is removed and the relevant code is changes so that the original tag uses the new code.